app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -164,25 +164,62 @@ public function sendSyncUrl(): void {
             return;
         }
 
-        $curl = new Varien_Http_Adapter_Curl();
-        $curl->setConfig([
-            'timeout'   => 10,
-        ]);
         $url = 'https://dashboard.webwinkelkeur.nl/webshops/sync_url';
         $data = json_encode([
             'webshop_id' => $helper->getShopId(),
             'api_key' => $helper->getApiKey(),
             'url' => Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_WEB) . 'webwinkelkeur/index/sync',
         ]);
+
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_TIMEOUT, 10);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
+        curl_setopt($ch, CURLOPT_FAILONERROR, true);
+        curl_setopt( $ch, CURLOPT_HTTPHEADER, array('Content-Type:application/json'));
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_POST, true);
+        curl_setopt($ch, CURLOPT_POSTFIELDS, $data);
```

Comment:
Same thing about error handling.  Given that this is the third spot where the same logic and most of the same options are needed, a helper class would be great.